lib/skinny_controllers/lookup/ensure_existence.rb

```diff
@@ -87,13 +78,7 @@ def policy_lookup(qualified_name)
     end
 
     def operation_lookup(qualified_name)
-      return if qualified_name.blank?
-      klass = qualified_name.safe_constantize
-
-      # Return if the constant exists, or if we can't travel
-      # up any higher.
-      return klass if klass
-      return unless qualified_name.scan(/::/).count > 1
```

Comment:
Cool,  yeah. If you want,  I think / thought 2 of the 3 methods could be refactoring?